app/lib/atom_serializer.rb

```diff
@@ -327,7 +331,9 @@ def add_namespaces(parent)
   end
 
   def serialize_status_attributes(entry, status)
-    append_element(entry, 'summary', status.spoiler_text) unless status.spoiler_text.blank?
+    if status.spoiler_text?
+      append_element(entry, 'summary', status.spoiler_text)
+    end
```

Comment:
I prefer the wrapped style, but not super strongly. Main change was using the query method on the atribute. I'd be fine changing to inline if desired.